Clinical-Genomics/cgweb
# loosen requirements and make yarn use the security patched versions

This PR updates some dependencies to security patched versions

How to test installation: 
1. Install branch on stage

Expected outcome: 
1. Installs w/o errors

How to test functionality:
1. Login
1. import an orderform

Expected outcome:
1. Login successful
1. The orderform is imported

Version: this is a patch since no functionality was added, only some security flaws were patched

- [x] code review: @dnil
- [x] deploy sign: @patrikgrenfeldt
